.changeset/sunny-cars-grin.md

```
@@ -0,0 +1,34 @@
+---
+"blocky-ui": minor
+---
+
+**Statistics Section**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)